From 405cc1e143bf3b9e87554a1e288d3c694f2b10b7 Mon Sep 17 00:00:00 2001
From: Flaviu <Flaviu@DESKTOP-HP>
Date: Thu, 2 Jan 2020 20:05:38 +0200
Subject: [PATCH 09/29] modifications

---
 Src/DirView.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Src/DirView.h b/Src/DirView.h
index ec2350675..0e62e2b7d 100644
--- a/Src/DirView.h
+++ b/Src/DirView.h
@@ -199,7 +199,7 @@ protected:
 	CSortHeaderCtrl m_ctlSortHeader;
 	CImageList m_imageList;
 	CImageList m_imageState;
-	CListCtrl *m_pList;
+	CListCtrl* m_pList;
 	std::unique_ptr<IListCtrl> m_pIList;
 	bool m_bEscCloses; /**< Cached value for option for ESC closing window */
 	bool m_bExpandSubdirs;
-- 
2.18.0.windows.1

