From bcb83af13ca10e9b07553bf6e6bd758dda515ee0 Mon Sep 17 00:00:00 2001
From: Flaviu <Flaviu@DESKTOP-HP>
Date: Thu, 2 Jan 2020 20:06:19 +0200
Subject: [PATCH 12/29] modifications

---
 Src/MainFrm.h | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/Src/MainFrm.h b/Src/MainFrm.h
index 08e255917..2b2c092ab 100644
--- a/Src/MainFrm.h
+++ b/Src/MainFrm.h
@@ -85,6 +85,7 @@ public:
 	LOGFONT m_lfDiff; /**< MergeView user-selected font */
 	LOGFONT m_lfDir; /**< DirView user-selected font */
 	static const TCHAR szClassName[];
+
 // Operations
 public:
 	HMENU NewDirViewMenu();
@@ -120,6 +121,7 @@ public:
 	void UpdateDocTitle();
 	void ReloadMenu();
 	DropHandler *GetDropHandler() const { return m_pDropHandler; }
+	const CTypedPtrArray<CPtrArray, CMDIChildWnd*>* GetChildArray() const { return &m_arrChild; }
 
 // Overrides
 	virtual void GetMessageString(UINT nID, CString& rMessage) const;
@@ -136,20 +138,18 @@ public:
 protected:
 	virtual ~CMainFrame();
 
-
 // Public implementation data
 public:
 	bool m_bFirstTime; /**< If first time frame activated, get  pos from reg */
 
 // Implementation data
 protected:
-
-
 	// control bar embedded members
 	CStatusBar  m_wndStatusBar;
 	CReBar m_wndReBar;
 	CToolBar m_wndToolBar;
 	CMDITabBar m_wndTabBar;
+	CTypedPtrArray<CPtrArray, CMDIChildWnd*> m_arrChild;
 
 	// Tweak MDI client window behavior
 	class CMDIClient : public CWnd
@@ -297,7 +297,12 @@ protected:
 	afx_msg void OnMRUs(UINT nID);
 	afx_msg void OnUpdateNoMRUs(CCmdUI* pCmdUI);
 	afx_msg void OnDestroy();
+	afx_msg void OnAccelQuit();
 	//}}AFX_MSG
+	afx_msg LRESULT OnChildFrameAdded(WPARAM wParam, LPARAM lParam);
+	afx_msg LRESULT OnChildFrameRemoved(WPARAM wParam, LPARAM lParam);
+	afx_msg LRESULT OnChildFrameActivate(WPARAM wParam, LPARAM lParam);
+	afx_msg LRESULT OnChildFrameActivated(WPARAM wParam, LPARAM lParam);
 	DECLARE_MESSAGE_MAP()
 
 private:
-- 
2.18.0.windows.1

