From 3347ce19a5ee6af716745152f0e7f3d3e7fafd8a Mon Sep 17 00:00:00 2001
From: Flaviu <Flaviu@DESKTOP-HP>
Date: Thu, 2 Jan 2020 20:06:58 +0200
Subject: [PATCH 15/29] modifications

---
 Src/MergeEditFrm.cpp | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/Src/MergeEditFrm.cpp b/Src/MergeEditFrm.cpp
index 682c5e059..77b9b75ab 100644
--- a/Src/MergeEditFrm.cpp
+++ b/Src/MergeEditFrm.cpp
@@ -279,7 +279,7 @@ BOOL CMergeEditFrame::DestroyWindow()
 	SavePosition();
 	SaveActivePane();
 	SaveWindowState();
-	return CMDIChildWnd::DestroyWindow();
+	return CMergeFrameCommon::DestroyWindow();
 }
 
 /**
@@ -315,7 +315,7 @@ void CMergeEditFrame::SaveActivePane()
 void CMergeEditFrame::OnClose() 
 {
 	// clean up pointers.
-	CMDIChildWnd::OnClose();
+	CMergeFrameCommon::OnClose();
 
 	GetMainFrame()->ClearStatusbarItemCount();
 }
@@ -388,7 +388,7 @@ void CMergeEditFrame::UpdateSplitter()
 void CMergeEditFrame::OnIdleUpdateCmdUI()
 {
 	UpdateHeaderSizes();
-	CMDIChildWnd::OnIdleUpdateCmdUI();
+	CMergeFrameCommon::OnIdleUpdateCmdUI();
 }
 
 void CMergeEditFrame::OnTimer(UINT_PTR nIDEvent) 
@@ -402,12 +402,12 @@ void CMergeEditFrame::OnTimer(UINT_PTR nIDEvent)
 	{
 		UpdateHeaderSizes();
 	}
-	CMDIChildWnd::OnTimer(nIDEvent);
+	CMergeFrameCommon::OnTimer(nIDEvent);
 }
 
 void CMergeEditFrame::OnMDIActivate(BOOL bActivate, CWnd* pActivateWnd, CWnd* pDeactivateWnd)
 {
-	CMDIChildWnd::OnMDIActivate(bActivate, pActivateWnd, pDeactivateWnd);
+	CMergeFrameCommon::OnMDIActivate(bActivate, pActivateWnd, pDeactivateWnd);
 
 	CMergeDoc *pDoc = GetMergeDoc();
 	if (bActivate && pDoc != nullptr)
-- 
2.18.0.windows.1

