From a264629f563b6fb43e028e63c7df23bba5f22b7f Mon Sep 17 00:00:00 2001
From: Flaviu <Flaviu@DESKTOP-HP>
Date: Thu, 2 Jan 2020 20:07:16 +0200
Subject: [PATCH 16/29] modifications

---
 Src/MergeFrameCommon.cpp | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/Src/MergeFrameCommon.cpp b/Src/MergeFrameCommon.cpp
index 7d533a3dd..b958a8d17 100644
--- a/Src/MergeFrameCommon.cpp
+++ b/Src/MergeFrameCommon.cpp
@@ -16,6 +16,7 @@ BEGIN_MESSAGE_MAP(CMergeFrameCommon, CMDIChildWnd)
 	//{{AFX_MSG_MAP(CMergeFrameCommon)
 	ON_WM_GETMINMAXINFO()
 	ON_WM_DESTROY()
+	ON_WM_MDIACTIVATE()
 	//}}AFX_MSG_MAP
 END_MESSAGE_MAP()
 
@@ -25,6 +26,12 @@ CMergeFrameCommon::CMergeFrameCommon(int nIdenticalIcon, int nDifferentIcon)
 	, m_bActivated(false)
 	, m_nLastSplitPos{0}
 {
+	::PostMessage(AfxGetMainWnd()->GetSafeHwnd(), WMU_CHILDFRAMEADDED, 0, reinterpret_cast<LPARAM>(this));
+}
+
+CMergeFrameCommon::~CMergeFrameCommon()
+{
+	::PostMessage(AfxGetMainWnd()->GetSafeHwnd(), WMU_CHILDFRAMEREMOVED, 0, reinterpret_cast<LPARAM>(this));
 }
 
 void CMergeFrameCommon::ActivateFrame(int nCmdShow)
@@ -106,3 +113,12 @@ void CMergeFrameCommon::OnDestroy()
 	CFrameWnd::OnDestroy();
 }
 
+void CMergeFrameCommon::OnMDIActivate(BOOL bActivate, CWnd* pActivateWnd, CWnd* pDeactivateWnd)
+{
+	// call the base class to let standard processing switch to
+	// the top-level menu associated with this window
+	CMDIChildWnd::OnMDIActivate(bActivate, pActivateWnd, pDeactivateWnd);
+
+	if (bActivate)
+		::PostMessage(AfxGetMainWnd()->GetSafeHwnd(), WMU_CHILDFRAMEACTIVATED, 0, reinterpret_cast<LPARAM>(this));
+}
-- 
2.18.0.windows.1

