From 340a2f5f2afe1b05b5848b97c755164986a47045 Mon Sep 17 00:00:00 2001
From: Flaviu <Flaviu@DESKTOP-HP>
Date: Thu, 2 Jan 2020 20:07:53 +0200
Subject: [PATCH 19/29] modifications

---
 Src/PropGeneral.cpp | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/Src/PropGeneral.cpp b/Src/PropGeneral.cpp
index e4df98642..a6231ba8a 100644
--- a/Src/PropGeneral.cpp
+++ b/Src/PropGeneral.cpp
@@ -41,17 +41,17 @@
  * @brief Constructor initialising members.
  */
 PropGeneral::PropGeneral(COptionsMgr *optionsMgr) 
-: OptionsPanel(optionsMgr, PropGeneral::IDD)
-, m_bScroll(false)
-, m_bSingleInstance(false)
-, m_bVerifyPaths(false)
-, m_bCloseWindowWithEsc(true)
-, m_bAskMultiWindowClose(false)
-, m_nAutoCompleteSource(0)
-, m_bPreserveFiletime(false)
-, m_bShowSelectFolderOnStartup(false)
-, m_bCloseWithOK(true)
-, m_pLoadLanguagesThread(nullptr)
+	: OptionsPanel(optionsMgr, PropGeneral::IDD)
+	, m_bScroll(false)
+	, m_bSingleInstance(false)
+	, m_bVerifyPaths(false)
+	, m_bCloseWindowWithEsc(false)	// default is false !
+	, m_bAskMultiWindowClose(false)
+	, m_nAutoCompleteSource(0)
+	, m_bPreserveFiletime(false)
+	, m_bShowSelectFolderOnStartup(false)
+	, m_bCloseWithOK(true)
+	, m_pLoadLanguagesThread(nullptr)
 {
 }
 
-- 
2.18.0.windows.1

