From 88171a3436cf6304efe2c39a3f84738f534b291e Mon Sep 17 00:00:00 2001
From: Takashi Sawanaka <sdottaka@users.sourceforge.net>
Date: Fri, 3 Jan 2020 10:41:56 +0900
Subject: [PATCH 22/29] Fix broken build (2)

---
 Src/Merge.rc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Src/Merge.rc b/Src/Merge.rc
index 10dd28fdb..56f4fa255 100644
--- a/Src/Merge.rc
+++ b/Src/Merge.rc
@@ -852,6 +852,7 @@ BEGIN
     "N",            ID_FILE_NEW,            VIRTKEY, CONTROL, NOINVERT
     "O",            ID_FILE_OPEN,           VIRTKEY, CONTROL, NOINVERT
     "P",            ID_FILE_PRINT,          VIRTKEY, CONTROL, NOINVERT
+    "Q",            ID_ACCEL_QUIT,          VIRTKEY, CONTROL, NOINVERT
     "S",            ID_FILE_SAVE,           VIRTKEY, CONTROL, NOINVERT
     "U",            ID_EDIT_LOWERCASE,      VIRTKEY, CONTROL, NOINVERT
     "U",            ID_EDIT_UPPERCASE,      VIRTKEY, SHIFT, CONTROL, NOINVERT
-- 
2.18.0.windows.1

